wbooks/src/app/constants.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Image } from 'react-native';
+import Colors from '@constants/colors';
+import backButton from '@assets/NavigationBar/ic_back.png';
+import headerBg from '@assets/General/bc_nav_bar.png';
+
+import styles from './style';
+
```

Comment:
It's ok but add a TODO to move this constants inside `AppNavigator` folder in the TabBar PR